icechunk/src/format/snapshot.rs

```diff
@@ -340,6 +340,7 @@ static ROOT_OPTIONS: VerifierOptions = VerifierOptions {
 
 impl Snapshot {
     pub const INITIAL_COMMIT_MESSAGE: &'static str = "Repository initialized";
+    pub const INITIAL_SNAPSHOT_ID: SnapshotId = SnapshotId::new([0; 12]);
```

Comment:
Let's make it something people can use to recognize this is an icechunk repo (we have some other ticket about that). Maybe we can put some bytes that when converted to the serialized id that goes in the object name, spells something similar to icechunk or something

Reply:
no, i don't think we need to check collisions, this is great